packages/flame_texturepacker/README.md

```diff
@@ -15,6 +15,7 @@ A flame plugin to import sprite sheets generated by <a href="https://github.com/
   <a title="Test" href="https://github.com/flame-engine/flame/actions?query=workflow%3Acicd+branch%3Amain"><img src="https://github.com/flame-engine/flame/workflows/cicd/badge.svg?branch=main&event=push"/></a>
   <a title="Discord" href="https://discord.gg/pxrBmy4"><img src="https://img.shields.io/discord/509714518008528896.svg"/></a>
   <a title="Melos" href="https://github.com/invertase/melos"><img src="https://img.shields.io/badge/maintained%20with-melos-f700ff.svg"/></a>
+  <a title="AI Assist" href="https://app.commanddash.io/agent/flame_engine"><img src="https://img.shields.io/badge/AI-Code%20Assist-EB9FDA.svg"></a>
```

Comment:
This should be removed, since it isn't in service anymore. (You probably based your commit off an old commit)

Reply:
This must have been from a failed rebase/merge @gnarhard, and it's still here :wink: